fmn/formatters.py

```diff
@@ -552,8 +553,13 @@ def _base_email(recipient=None, messages=None):
                 _log.exception('fedmsg.meta.msg2usernames failed to handle %r', msg)
             try:
                 for package in fedmsg.meta.msg2packages(msg, **config.app_conf) or []:
-                    email_message.add_header('X-Fedmsg-Package', package.encode('utf-8'))
+                    packages.add(package)
             except Exception:
                 _log.exception('fedmsg.meta.msg2packages failed to handle %r', msg)
 
+        for pkg in list(packages)[:10]:
+            email_message.add_header('X-Fedmsg-Package', pkg.encode('utf-8'))
+        if len(packages) > 10:
```

Comment:
I'd even always include this one, then we're consistent.

Reply:
Done, thanks for the hint!